services/run-cloudbackup.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Starts creating a zfs snapshot for the cloud backup
+ConditionPathExists=/etc/protonet/soul/enabled
```

Comment:
alright. I figured soul/enabled was the general trigger for soul and everything connected. I'll change it as suggested. do i need to enable this somewhere then by default or will you take care. and if I have to, could you point me into the general direction?